viadot/sources/outlook.py

```diff
@@ -0,0 +1,126 @@
+from .base import Source
+from O365 import Account
+import pandas as pd
+import datetime
+from typing import Any, Dict, List
+from ..config import local_config
+
+
+class Outlook(Source):
+    def __init__(
+        self,
+        mailbox_name: str,
+        start_date: str = None,
+        end_date: str = None,
+        credentials: Dict[str, Any] = None,
+        extension_file: str = ".csv",
+        limit: int = 10000,
+        request_retries: int = 10,
+        *args: List[Any],
+        **kwargs: Dict[str, Any],
+    ):
+        """Outlook connector build for fetching Outlook API source.
```

Comment:
It's a small thing but please check your docstring format and unify. You can check how they are built in other files in viadot. 
Here you have:
```
1) 
"""Something...
"""
OR
2) 
"""
Something
"""
````